src/drivers/imx/esai.c

```diff
@@ -99,9 +100,13 @@ static inline int esai_set_config(struct dai *dai,
 				 struct sof_ipc_dai_config *config)
 {
 	uint32_t xcr = 0, xccr = 0, mask;
+	struct esai_pdata *sai = dai_get_drvdata(dai);
 
```

Comment:
Lets name this var esai because sai is another type of DAI for imx.

Reply:
done